services/traction/api/core/config.py

```diff
@@ -88,6 +88,9 @@ class GlobalConfig(BaseSettings):
 
     DEFAULT_PAGE_SIZE: int = os.environ.get("DEFAULT_PAGE_SIZE", 10)
 
+    # Allow CORS from a comma-separated list of origins
+    TRACTION_CORS_URLS: str = os.environ.get("TRACTION_CORS_URLS", "")
```

Comment:
Think we should add this to env-example so people can just set it when they need it after copying over the example.
See how we do SHOWCASE_CORS_URLS in .env-example

Reply:
Good thinking.